README.md

```diff
@@ -67,6 +67,12 @@ console.log(reactElementToJSXString(<div a="1" b="2">Hello, world!</div>));
 
   Just return the name you want for the provided ReactElement, as a string.
 
+**options.showBooleans: boolean, default false**
```

Comment:
Can we rename it to `useBooleanShorthandSyntax`? Long but explicit, showBooleans is not precise enough I believe